VERSION

```diff
@@ -1 +1 @@
-0.5.2
+0.5.3
```

Comment:
could be 0.6.0 bump

Reply:
or should, I think

Reply:
Change is that major? ;-)

Reply:
no, minor :) major would increment the first digit.

minor is for added functionality
the last digit is for patches / bug fixes